packages/http-specs/specs/encode/bytes/main.tsp

```diff
@@ -229,6 +229,9 @@ namespace RequestBody {
     """)
   @post
   op default(
+    @header
```

Comment:
I think just need to update the doc above the default cases

Reply:
I think this test needs to change actually to show that by default this is a binary payload

Reply:
changed.